### PR TITLE
refactor(choice.html): change layout of choice

### PR DESCRIPTION
**Pull request by arminsebastian**

### CODE REVIEW

File: client/styles/styles.css [hunk 1]
```diff
@@ -405,6 +405,21 @@ body > div:first-of-type > div:first-of-type {
 .col-left {
   padding: 0 15px;
   float: left;
+<<<<<<< HEAD
+}
```
Comment (darrinmn9) [inline client/styles/styles.css:432]: _merge conflicts_

File: client/styles/styles.css [hunk 2]
```diff
@@ -405,6 +405,21 @@ body > div:first-of-type > div:first-of-type {
...
+
+.col-left-hero {
```
Comment (tats-tagawa) [inline client/styles/styles.css:434]: I can't find a tag with a class name of `.col-left-hero` anywhere. Which element is it for?

I significantly changed the column structure in my pull request and changing the list column may have significant effect on it. However, I am still debating whether mine should be merged into dev or not. We can discuss tomorrow.

**Comment by tats-tagawa** [comment]
The three images' file size ranges from 2MB to 3.5MB. This is too big and will take time to load, especially from a heroku server. Since it is going to blurred out anyway, I recommend lowering the quality/size significantly so the file size is about 100 - 300KB. 

Solved

File: client/styles/styles.css (hunk 2 above)
Comment (arminsebastian) [inline client/styles/styles.css:434]: Deleted the unused class

File: client/styles/styles.css (hunk 1 above)
Comment (arminsebastian) [inline client/styles/styles.css:432]: deleted

**Pull request closed without merging**